cypress/integration/verify/ui.spec.js

```diff
@@ -0,0 +1,53 @@
+/// <reference types="Cypress" />
+
+// `verify` function template UI-only tests
+
+const testNumber = "5555555555";
+const testEmail = "test@example.org";
+
+describe('verify Template UI', () => {
+    beforeEach(() => {
+        cy.visit('/verify/index.html');
+    });
+
+    it('should identify itself as the Verify example', () => {
+        cy.contains('Twilio Verify');
+    });
+
+    it('should accept a number to verify via SMS by default', () => {
+        cy
+            .get('[data-cy=input-phone]')
+            .type(testNumber)
+            .get('[data-cy=submit]')
+            .click();
+
+        // NOTE: This response will change when Cypress integration tests are incorporated
+        cy.contains('Error');
+    });
+
+    it('should support verifying via a phone call', () => {
+        cy
+            .get('[data-cy=radio-call]')
+            .click()
+            .get('[data-cy=input-phone]')
+            .type(testNumber)
+            .get('[data-cy=submit]')
+            .click();
+
+        // NOTE: This response will change when Cypress integration tests are incorporated
+        cy.contains('Error');
+    });
+
+    it('should support verifying via email', () => {
+        cy
+            .get('[data-cy=radio-email]')
+            .click()
+            .get('[data-cy=input-email]')
+            .type(testEmail)
+            .get('[data-cy=submit]')
+            .click();
+
+        // NOTE: This response will change when Cypress integration tests are incorporated
+        cy.contains('Error');
+    });
```

Comment:
Something I think i failed to go over with you, there are `<!-- APP_INFO -->` or `<!-- EDIT_CODE -->` comments in most of the index.html files. They are triggers that we use on the backend to inject the links to edit the deployed code directly in the console.

It would be good to make sure that is captured in our tests

Reply:
I wrote up a quick test for the verify template that uses a DOM `TreeWalker` to ensure index.html only contains an `EDIT_CODE` comment with no `APP_INFO`. Right now it accepts more than one `EDIT_CODE`; is there any case in which there will be more than one `EDIT_CODE` tag?